.github/workflows/test.yml

```diff
@@ -22,8 +22,6 @@ jobs:
         os: [ ubuntu-latest, windows-latest, macos-latest ]
         python-version: [ "3.13" ]
         include:
-          - os: ubuntu-22.04
-            python-version: "3.7"
```

Comment:
This will be done properly in https://github.com/fastapi/typer/pull/830, but I don't want the CI here to be red just because of Python 3.7.